tests/test_validators.py

```diff
@@ -733,6 +733,18 @@ def check_something(cls, v):
     assert val_calls == [None, 'hello']
 
 
+def test_required_optional():
+    class Model(BaseModel):
+        nullable: Optional[int] = Field(...)
```

Comment:
```suggestion
        nullable: Optional[int] = ...
```
fine to also add a `Field(...)` test, but this is the default case.

Reply:
Sure!